server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -13,7 +13,10 @@
 module Fusor
   class Api::V21::DeploymentsController < Api::V2::DeploymentsController
 
-   before_filter :find_deployment, :only => [:destroy, :show, :update, :deploy]
+    before_filter :find_deployment, :only => [:destroy, :show, :update, :deploy]
+    skip_before_filter :verify_authenticity_token
+
+    rescue_from Encoding::UndefinedConversionError, :with => :ignore_it
```

Comment:
not sure else how to save this strange error
```
Encoding::UndefinedConversionError: "\xBA" from ASCII-8BIT to UTF-8
```
It my searches, It's a ruby issue, not JS